commands/ls.go

```diff
@@ -59,7 +59,7 @@ func runLs(dockerCli command.Cli, in lsOptions) error {
 	}
 
 	w := tabwriter.NewWriter(dockerCli.Out(), 0, 0, 1, ' ', 0)
-	fmt.Fprintf(w, "NAME/NODE\tDRIVER/ENDPOINT\tSTATUS\tBUILDKIT\tPLATFORMS\n")
+	fmt.Fprintf(w, "NAME/NODE\tDRIVER/ENDPOINT\tSTATUS\tBKVERSION\tPLATFORMS\n")
```

Comment:
Hm, I think this is less readable than before - for novice users, I don't think it's clear what `BK` stands for (and it's definitely a lot harder to google for).